packages/lib-content/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@zooniverse/content",
+  "description": "A library of components used in Zooniverse content pages",
+  "license": "Apache-2.0",
+  "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
+  "version": "0.0.1",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./*": {
+      "import": "./dist/esm/*/index.js",
+      "require": "./dist/cjs/*/index.js"
+    }
+  },
+  "sideEffects": false,
+  "repository": "https://github.com/zooniverse/front-end-monorepo.git",
+  "bugs": "https://github.com/zooniverse/front-end-monorepo/issues",
+  "scripts": {
+    "build": "yarn build:cjs && yarn build:es6",
+    "build:cjs": "babel ./src/ --out-dir ./dist/cjs --copy-files --no-copy-ignored --ignore 'src/**/*.spec.js' --ignore 'src/**/*.stories.js'",
+    "build:es6": "BABEL_ENV=es6 babel ./src/ --out-dir ./dist/esm --copy-files --no-copy-ignored --ignore 'src/**/*.spec.js' --ignore 'src/**/*.stories.js'",
+    "test": "mocha --config ./test/.mocharc.json ./.storybook/specConfig.js \"./src/**/*.spec.js\"",
+    "test:ci": "mocha --config ./test/.mocharc.json ./.storybook/specConfig.js --reporter=min \"./src/**/*.spec.js\"",
+    "storybook": "storybook dev -p 6005",
+    "build-storybook": "storybook build"
```

Comment:
Will this add @zooniverse/content to https://zooniverse.github.io/front-end-monorepo/ ? And double-checking if that is intended or not?

Reply:
I think it will! I do want a new Storybook for this lib deployed eventually.